jruby-gradle-base-plugin/src/test/groovy/com/github/jrubygradle/JRubyPluginTest.groovy

```diff
@@ -31,7 +31,7 @@ class JRubyPluginTest {
 
     @Test
```

Comment:
Just wondering why we have not yet converted this to SpockFw...

Never mind too much, not needed for this PR.